hack/dev-setup-register-gardener

```diff
@@ -128,6 +164,7 @@ spec:
   service:
     name: gardener-apiserver
     namespace: garden
+$APISERVICE_PORT_STRING
```

Comment:
```suggestion
    port: $APISERVER_SERVICE_PORT
```

Reply:
Can you explain, why this is needed?
I didn't quite get it...

Reply:
if it is docker, the port won't be set (docker does not support port since it runs on max 1.14)

Reply:
Alright. Got it.